CLI/library.php

```diff
@@ -308,6 +308,55 @@ function concatenateTracks($first, $second, $output, $remove_sources = true)
     }
 }
 
+/**
+ * This function invokes SoX to place the first track before the intermediate and last tracks and return that as the output file, optionally, removing the sources once it's done
+ *
+ * @param path    $first          The first file to use
+ * @param path    $last           The last file to use
+ * @param path    $intermediate   The intermediate file - with {n} token to substitute in the cycled value of $num_tracks
+ * @param int     $num_tracks     The number of intermediate files to cycle through
+ * @param path    $output         The output location of the combined two files
+ * @param boolean $remove_sources Whether the script should remove the two source files.
+ *
+ * @return boolean Success or Failure
+ */
+function concatenateMultiTracks($first, $last, $intermediate, $num_tracks, $output, $remove_sources = true)
+{
+
+//    print ('first: ' . $first . ' : ');
+//    print ('last: ' . $last . ' : ' );
+//    print ('intermediate: ' . $intermediate . ' : ');
+//    print ('num_tracks: ' . $num_tracks . ' : ');
+//    print ('output: ' . $output );
```

Comment:
Let's remove these debugging lines, as they're not really needed any more?

Reply:
Will be in next commit